### DISCUSSION TITLE
Added Bark Plugin

### DISCUSSION THREAD

**Discussion by TheCompAce** [TheCompAce/Auto-GPT-Powershell#12] [Show and tell]
This uses "Bark" to create speech from Text. This plugin ask the GPT to add "<BARK dest='filename.wav' sex='MALE/FEMALE' voice='0-9'>text</BARK>" tags to the response, then it takes those and saves them to the "session_[time]" folder in a "Voice" folder.

**Comment by Kallamamran** [comment]
![image](https://user-images.githubusercontent.com/25354644/236036639-40fca7e5-0c24-439a-952c-130aab448c6e.png)
It seems Bark doesn't want to play or rather... It messes things up even if I have disbled it.

**Reply by kreutzi** [reply]
did you fix it? getting the same errors
